specs/specs.go

```diff
@@ -98,14 +101,32 @@ type StringAssert struct {
 	NotContains *string `json:"notContains,omitempty"`
 }
 
-func LoadSuite(path string) *Suite {
-	data, err := ioutil.ReadFile(path)
+type TemplateSpec struct {
+	Vars map[string]string
+	Env  map[string]string
+}
+
+func getEnvs() map[string]string {
+	envs := make(map[string]string)
+	for _, element := range os.Environ() {
+		splits := strings.Split(element, "=")
```

Comment:
`SplitN(element, "=", 2)`, to properly handle case where `=` is the value content.